packages/react/src/components/chooser-button/styles/choose.ts

```diff
@@ -1,60 +1,57 @@
-            min-height: 3rem;
-            padding: var(--spacing-1x);
-            text-align: center;
-            transition: all 0.25s ease-in-out;
-
-            &:hover {
-                background: ${theme.component['chooser-hover-background-color']};
-                border-color: ${theme.component['chooser-hover-border-color']};
-                color: ${theme.component['chooser-hover-text-color']};
-            }
-
-            input[type="checkbox"]:checked + &,
-            input[type="radio"]:checked + & {
-                background: ${theme.component['chooser-selected-background-color']};
-                border-color: ${theme.component['chooser-selected-border-color']};
-                color: ${theme.component['chooser-selected-text-color']};
-            }
-
-            ${focus({ theme }, true, 'input[type="checkbox"]:focus + &, input[type="radio"]:focus + &')}
-
-            input[type="checkbox"]:disabled + &,
-            input[type="radio"]:disabled + & {
-                background: ${theme.component['chooser-disabled-background-color']};
-                border-color: ${theme.component['chooser-disabled-border-color']};
-                color: ${theme.component['chooser-disabled-text-color']};
-            }
-
-            b {
-                font-size: 1.5rem;
-                font-weight: var(--font-normal);
-                line-height: 2.5rem;
-            }
-
-            /* For future support of SVG icons in button */
-            svg {
-                color: inherit;
-                height: 3rem;
-                width: 3rem;
-            }
-        `}
+    --border-radius: 8px;
+
+    align-items: center;
+    background: ${({ theme }) => theme.component['chooser-background-color']};
+    border: 1px solid ${({ theme }) => theme.component['chooser-border-color']};
+    border-radius: var(--border-radius);
+    box-sizing: border-box;
+    color: ${({ theme }) => theme.component['chooser-text-color']};
+    display: flex;
+    font-size: 1rem;
+    justify-content: center;
+    letter-spacing: 0;
+    line-height: 1.5rem;
+    min-height: 3rem;
+    padding: var(--spacing-1x);
+    text-align: center;
+    transition: all 0.25s ease-in-out;
+
+    &:hover {
+        background: ${({ theme }) => theme.component['chooser-hover-background-color']};
+        border-color: ${({ theme }) => theme.component['chooser-hover-border-color']};
+        color: ${({ theme }) => theme.component['chooser-hover-text-color']};
+    }
+
+    input[type='checkbox']:checked + &,
+    input[type='radio']:checked + & {
+        background: ${({ theme }) => theme.component['chooser-selected-background-color']};
+        border-color: ${({ theme }) => theme.component['chooser-selected-border-color']};
+        color: ${({ theme }) => theme.component['chooser-selected-text-color']};
+    }
+
+    ${({ theme }) => focus({ theme }, { selector: ', input[type="checkbox"]:focus + &, input[type="radio"]:focus + &' })}
```

Comment:
Le selector n'est pas propre, c'est comme un hack pour contourner les lacunes du param "selector".

Le param selector n'est pas clair, dans la function c'est vu comme un childs selector (il n'y a rien dans la function qui m'indique autrement, sinon il faudrait faire des params separe ou documenter l'usage).
```
&:${focusTypeSelector} ${baseSelector} {
```

Avec la virgule, tu append d'autre selectors, mais le `&:focus` est encore la, donc ca laisse trainer des unwanted css aussi.

Aussi le ":not" ne functionne pas avec ca, ca donnerait
```
&:not(:focus), 
input[type="checkbox"]:focus + &, 
input[type="radio"]:focus + & {
```

Je me demande a quel point la transition est necessaire, ca ajoute trop de complexite. 

Faidrais p-t avoir une autre facon d'utiliser juste les styles sans les selectors, genre
```
input[type="checkbox"]:not(:focus) + &, 
input[type="radio"]:not(:focus) + & {
  ${unfocusedStyles(...)}
}

input[type="checkbox"]:focus + &, 
input[type="radio"]:focus + & {
  ${focusedStyles(...)}
}
```